src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = {};
+  const arrayOfItems = sourceString
+    .split(';') // ['ключ : свойство','ключ:свойство']
+    .filter((item) => item.includes(':')) // убираем  пустые элементы
+    .map((elem) => elem.split(':')); //[[ключ, свойство], [ключ, свойство]]
```

Comment:
The variable name `arrayOfItems` could be improved to reflect the contents more accurately. Avoid generic names like `array` or `arrayOfItems` as per the [CODE STYLE] checklist. A more descriptive name could be `styleRules` or `cssDeclarations`.